challenge-02/README.md

```diff
@@ -58,8 +73,27 @@ Crie uma função com as seguintes características:
 5. Se nenhum argumento for passado, retorne o valor booleano `false`.
 6. E ainda, se nenhuma das condições acima forem atendidas, retorne `null`.
 */
-?
+function outraQualquer(x, y, z){
+	if(x != null){
+		if(y != null && z == null){
+			return x + y;
+		}else if(z != null){
+			return (x + y) / z;
+		}
+		return x;
+	}else {
+		return false;
+	}
+	return null;
+}
```

Comment:
-1
Aqui é o mesmo esquema da função anterior: é necessário usar o `undefined` para verificar se o valor não foi mesmo passado.. no caso, eu poderia passar `null` como algum argumento, o que faria quebrar essa lógica :)

Outra dica: sempre que for fazer comparação, utilize o operador estrito: `===` :)